src/applications/edu-benefits/sass/edu-benefits.scss

```diff
@@ -171,6 +161,6 @@
 
 div[data-chapter="benefitSelection"]{
```

Comment:
is this class needed for both 1995 and 10203?
or can it be removed since it is already in `src/applications/edu-benefits/10203/sass/10203_stem.scss` ?

Reply:
Looks like we can go ahead and remove that since it is redundant.